plugins/file/plugin.go

```diff
@@ -7,6 +7,7 @@ import (
 	"io/ioutil"
 	"net"
 	"strings"
+	"time"
```

Comment:
I would leave the `file` plugin unmodified, and simply focus on the `range` one.

Reply:
Do you mean reverting file plugin to its original state?

Reply:
With an empty handler for ipv4?

Reply:
Yes, I would leave it as it was before, without the lease time config in it since it is aimed for static allocation. There is not much sense to store the lease time in the file.

Reply:
You can add feature parity with the v6 one but I wouldn't add lease time to it.

Reply:
So then the range plugin would have a different lease file and range and plugin should not be running on the same server?

Reply:
If so, then the range plugin should check the leased IP addresses as file plugin did before this commit?

Reply:
You can run them in the same server, I would use a different file for static and dynamic allocation. I would keep the plugins independent so that the file format change won't break both plugins.

Reply:
> You can add feature parity with the v6 one but I wouldn't add lease time to it.

I'm not sure I understood you, could you elaborate?

Reply:
> You can run them in the same server, I would use a different file for static and dynamic allocation. I would keep the plugins independent so that a file format change won't break both plugins.

But then it could happen that the same IP address is leased to different devices.

Reply:
> I'm not sure I understood you, could you elaborate?

When I said "You can add feature parity with the v6 one but I wouldn't add lease time to it." I was replying to your "With an empty handler for ipv4?" question.

You can go ahead and modify the `file` plugin if you want to add the missing functionality but I wouldn't add a lease time to it. I would focus on the `range` plugin at this point.

Reply:
Ok, I got it now, thank you.

Reply:
> EDIT: I guess you could prevent that so the static IP addresses are not in the range of the range plugin.

Right, you can have a pool of IPs for each thing.